spec/index.html

```diff
@@ -7410,6 +7410,127 @@ <h5>TZ</h5>
             </div>
           </section>
         </section>
+
+        <!-- QT -->
+        <section id="func-quoted-triples">
+          <h4>Functions on Quoted Triples</h4>
+          <section id="func-triple">
+            <h5>TRIPLE</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">quoted triple</span>  <span class="operator">TRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">subj</span>, <span class="type RDFterm">RDF term</span> <span class="name">pred</span>, <span class="type RDFterm">RDF term</span> <span
+class="name">obj</span>)
+            </pre>
+            <pre class="query nohighlight">
+                  <span class="operator">&lt;&lt;</span> subj pred obj <span class="operator">&gt;&gt;</span>
+            </pre>
+            <p>
+              If the 3-tuple (<code style="color:black">subj</code>, 
+              <code style="color:black">pred</code>, 
+              <code style="color:black">obj</code>)
+              is an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>
+              (that is, <code style="color:black">subj</code> is an
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a> or a 
+              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>,
+              <code style="color:black">pred</code> is an 
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              and <code style="color:black">obj</code> is an
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a> or a 
+              <a data-cite="RDF12-CONCEPTS#dfn-literal">literal</a>)
+              the function returns a quoted triple with these three elements.
+              Otherwise, the function raises an error.
+            </p>
+
+            <p>
+              As a shorthand notation, the <code>TRIPLE</code> function 
+              can also be written in the form of a
+              <a href="#rExprQuotedTriple">quoted triple expression</a>
+              using <code>&lt;&lt;</code> and <code>&gt;&gt;</code>.  There is a
+              syntax limitation to this shorthand form because the three elements of
```

Comment:
```suggestion
              syntax limitation to this shorthand form — the three elements of
```

Reply:
Thank you for the comment. 
"because" is used quite a few other places in the document.

Reply:
I must not be understanding something here.

Is the syntax limitation of this shorthand form _caused by_ `the three elements of the quoted triple expression can only be variables and directly written RDF terms, not arbitrary expressions`, in which case your `because` is fine (but I'd like to also see description of the limitation which is so caused)?

Or is the syntax limitation of this shorthand form _the fact that_ `the three elements of the quoted triple expression can only be variables and directly written RDF terms, not arbitrary expressions`, which is my understanding, and the reason I replaced `because` with `—`.

Reply:
I agree with @TallTed here, because the syntax limitation is indeed the fact that "the three elements of the quoted triple expression can only be variables [...]."

What I usually do in such cases is to use a colon. Hence, my suggested change for this part is as follows.
```suggestion
              syntax limitation to this shorthand form: the three elements of
```

Reply:
A colon would also work.